man/adoc/bpftrace.adoc

```diff
@@ -1756,6 +1756,47 @@ This function can only be used by functions that are allowed to, these functions
 
 Removes a watchpoint
 
+=== skboutput
+
+.variants
+* `uint32 skboutput(const string path, struct sk_buff *skb, uint64 length, const uint64 offset)`
+
+*Kernel* 5.5
+
+*Helper* bpf_skb_output
+
+Write sk_buff `skb` data to a PCAP file in the `path`, starting from `offset` to `offset` + `length`.
```

Comment:
is this offset in the pcap data file or offset in the struct?

Reply:
When bpf_skb_output is called on `dev_queue_xmit`'s context, the `skb`'s data section is prefixed by 14 bytes of ethernet header. On the context of `napi_gro_receive`, there is no ethernet header, only IP and upper layer headers.

So I added the `offset` arg to indicate how much bytes should we ignore from the beginning of skb's data, to make a legit Raw IP format pcap file. 

Is it okay to have this offset arg?  @fbs 

I'll updated the doc to make it clear.